sql/core/src/test/resources/sql-tests/results/ansi/interval.sql.out

```diff
@@ -117,6 +117,75 @@ struct<(INTERVAL '02' SECOND * NULL):interval day to second>
 NULL
 
 
+-- !query
+select interval '2 year' / 0
+-- !query schema
+struct<>
+-- !query output
+java.lang.ArithmeticException
+/ by zero
+
+
+-- !query
+select interval '2 year' / null
+-- !query schema
+struct<(INTERVAL '2' YEAR / NULL):interval year to month>
```

Comment:
I wonder why the type is `interval year to month` but not `interval year`

Reply:
good point. This is probably something we should fix.

Reply:
I am ok to fix this in a separate PR. @gengliangwang Could you open a sub-task in https://issues.apache.org/jira/browse/SPARK-27790, please.

Reply:
This is the data type of `DivideYMInterval`, which is the default YM interval type.

Reply:
It's to have a wider data type as the division result. Just like the following
```
scala> spark.sql("select 1 /null")
res0: org.apache.spark.sql.DataFrame = [(1 / NULL): double]
```

Reply:
Got it. Thanks.